library/core/src/main/java/com/google/android/exoplayer2/video/VideoFrameReleaseTimeHelper.java

```diff
@@ -100,6 +101,43 @@ public void disable() {
     if (useDefaultDisplayVsync) {
       vsyncSampler.removeObserver();
     }
+    unregisterDisplayListener();
```

Comment:
Given registerDisplayListener is in the if block in enable, this should probably be in the if block in disable also.